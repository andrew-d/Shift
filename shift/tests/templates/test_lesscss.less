@variable: 10px;

.class {
    margin: @variable;
}
