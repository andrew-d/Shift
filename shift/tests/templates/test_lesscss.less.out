.class { margin: 10px; }